src/components/Form.js

```diff
@@ -340,10 +344,13 @@ class Form extends Component {
             }
 
             if (input) {
-                input.focus();
+                setTimeout(() => input.focus(), 1000);
             }
         }
+    }
 
+    handleSubmit = (e, handleSubmit) => {
+        this.scrollToInvalidField();
```

Comment:
try move to Formspy https://github.com/experium/jobot-forms/blob/595c3e9ab0d48b6e4e6eb84d24427add503e7340/src/components/Form.js#L386

and remove setTimeout